Model/Config/Adminhtml/MerchantAccounts.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Model\Config\Adminhtml;
+
+class MerchantAccounts extends \Magento\Config\Block\System\Config\Form\Field
+{
+    protected $_template = 'Adyen_Payment::config/required_settings.phtml';
+
+    public function render(\Magento\Framework\Data\Form\Element\AbstractElement $element)
```

Comment:
Can we add a return type in the functions of this class?

Reply:
interface doesn't declare a return type